src/manifest.json

```diff
@@ -47,5 +66,8 @@
     "storage",
     "unlimitedStorage"
   ],
-  "host_permissions": ["http://*/*", "https://*/*"]
+  "host_permissions": [
+    "http://*/*",
+    "https://*/*"
+  ]
```

Comment:
why this diff?

Reply:
auto formatting probably, will check my editor settings

Reply:
:check: reverted, will update my neovim config tonight :)